resources/views/livewire/terminal/index.blade.php

```diff
@@ -8,27 +8,32 @@
         <x-helper
             helper="If you're having trouble connecting to your server, make sure that the port is open.<br><br><a class='underline' href='https://coolify.io/docs/knowledge-base/server/firewall/#terminal' target='_blank'>Documentation</a>"></x-helper>
     </div>
-    <div>
-        <form class="flex flex-col gap-2 justify-center xl:items-end xl:flex-row"
-            wire:submit="$dispatchSelf('connectToContainer')">
-            <x-forms.select id="server" required wire:model.live="selected_uuid">
-                @foreach ($servers as $server)
-                    @if ($loop->first)
-                        <option disabled value="default">Select a server or container</option>
-                    @endif
-                    <option value="{{ $server->uuid }}">{{ $server->name }}</option>
-                    @foreach ($containers as $container)
-                        @if ($container['server_uuid'] == $server->uuid)
-                            <option value="{{ $container['uuid'] }}">
-                                {{ $server->name }} -> {{ $container['name'] }}
-                            </option>
+    <div x-init="$wire.loadContainers()">
+        @if ($isLoadingContainers)
+            <div class="pt-1">
+                <x-loading text="Loading servers and containers..." />
+            </div>
+        @else
+            <form class="flex flex-col gap-2 justify-center xl:items-end xl:flex-row"
+                wire:submit="$dispatchSelf('connectToContainer')">
+                <x-forms.select id="server" required wire:model.live="selected_uuid">
+                    @foreach ($servers as $server)
+                        @if ($loop->first)
+                            <option disabled value="default">Select a server or container</option>
                         @endif
+                        <option value="{{ $server->uuid }}">{{ $server->name }}</option>
+                        @foreach ($containers as $container)
+                            @if ($container['server_uuid'] == $server->uuid)
+                                <option value="{{ $container['uuid'] }}">
+                                    {{ $server->name }} -> {{ $container['name'] }}
+                                </option>
+                            @endif
+                        @endforeach
                     @endforeach
-                @endforeach
-            </x-forms.select>
-            <x-forms.button type="submit">Connect</x-forms.button>
-        </form>
-        <livewire:project.shared.terminal />
+                </x-forms.select>
+                <x-forms.button type="submit">Connect</x-forms.button>
+            </form>
+            <livewire:project.shared.terminal />
```

Comment:
was just testing next local and terminal wouldnt work, so,
`<livewire:project.shared.terminal />` should go after `@endif`
or should patch the js of terminal

cause is the new conditional rendering is hiding #terminal-container element and alpine cant be attached to on init

Reply:
Oh, nice finding! Thank you! I will fix it!

Reply:
Fixed in this branch! Thanks again @LEstradioto